### DISCUSSION TITLE
No module named atri_utils

### DISCUSSION THREAD

**Discussion by rohan1407tech** [Atri-Labs/atrilabs-engine#589] [Help: Installation & Start] [answered]
On giving the command **atri start** it show downloading atri engine and then shows error and hence the framework doesn't start
I have added pipenv to the path as well using the instructions provided

**Comment by darshitac11** [comment]
@rohan1407tech - please share screenshot of your terminal so that we can better understand your issue.

**Reply by rohan1407tech** [reply]
I resolved that problem but while running the editor it throws another problem i am sharing the SS of my terminal window
![image](https://user-images.githubusercontent.com/75736670/209330687-a573f6be-c03f-45b4-a7f0-1151541a36fc.png)

**Reply by darshitac11** [reply]
It has already been answered here: https://github.com/Atri-Labs/atrilabs-engine/discussions/437

**Comment by darshitac11** [comment, marked as answer]
It has already been answered here: https://github.com/Atri-Labs/atrilabs-engine/discussions/437
